### PR TITLE
fixing jshint score and deleted globalstrict in .jshint as it is depr…

### PR DESCRIPTION
**Pull request by tldavies**
…ecated

### CODE REVIEW

**Comment by tldavies** [comment]
@rolandpoulter @DavidjohnBlodgett @RackHD/corecommitters

**Comment by heckj** [comment]
+1

File: lib/jobs/install-os.js [hunk 1]
```diff
@@ -142,6 +117,30 @@ function installOsJobFactory(
...
+        //The value from shared context will override the value in self options.
+        self.options = _.assign(self.options, context.repoOptions);
+
+        _validateOptions.call(self);
+        _convertOptions.call(self);
+        _encryptPassword.call(self);
+    }
+    util.inherits(InstallOsJob, BaseJob);
+
+    /**
```
Comment (DavidjohnBlodgett) [inline lib/jobs/install-os.js]: why did we need to move this code?

Comment (benbp) [inline lib/jobs/install-os.js]: Yeah, the constructor should conventionally be at the top of the file.

**Comment by benbp** [comment]
-1 because of the constructor move.

**Comment by tldavies** [comment]
jshint yelled cause we are using functions that arent defined yet. So I moved the function before we use them.

File: lib/jobs/install-os.js (hunk 1 above)
Comment (tldavies) [inline lib/jobs/install-os.js]: jshint yelled cause we are using functions that arent defined yet. So I moved the functions before we use them.

**Comment by DavidjohnBlodgett** [comment]
@benbp sounds like we can't get around it to me.

File: lib/jobs/install-os.js [hunk 2]
```diff
@@ -46,16 +45,19 @@ function installOsJobFactory(
         //The value from shared context will override the value in self options.
         self.options = _.assign(self.options, context.repoOptions);
 
-        _validateOptions.call(self);
-        _convertOptions.call(self);
-        _encryptPassword.call(self);
+        InstallOsJob._validateOptions();
```
Comment (benbp) [inline lib/jobs/install-os.js]: Should be `this`

**Comment by DavidjohnBlodgett** [comment]
:+1:

**Comment by tldavies** [comment]
@benbp fixed :-)

**Comment by benbp** [comment]
:+1:

**Pull request merged**